contrib/test.sh

```diff
@@ -2,35 +2,51 @@
 
 FEATURES="serde"
 
-# Use toolchain if explicitly specified
-if [ -n "$TOOLCHAIN" ]
-then
-    alias cargo="cargo +$TOOLCHAIN"
-fi
-
 # Pin dependencies as required if we are using MSRV toolchain.
-if cargo --version | grep "1\.41"; then
+if cargo --version | grep "1\.48"; then
```

Comment:
I have suspect toolchain 1.48 is not really running... Locally with 1.449 I have errors on `which` and I think it needs at least:

```
cargo update -p which --precise 4.3.0
```

Reply:
Can you post your exact invocation? `cargo +1.48.0 test --all-features --all-targets` runs clean for me with only the serde pin.

Reply:
Ok, if I do `cargo +stable update` then it pulls in a version of `once_cell` thatt 1.48.0 doesn't recognize, and which gives me an error message blaming `which` for some reason. But if I do `cargo +1.48.0 update` then this doesn't happen.

So I think we're fine.